tests/docker/ducker-rp

```diff
@@ -254,9 +254,9 @@ docker_run() {
     must_do -v ${DOCKER} run --privileged \
         -d -t -h "${node}" --network ducknet "${expose_ports}" \
         --memory=${docker_run_memory_limit} --memory-swappiness=1 \
-        -v "${v_dir}/build:${v_dir}/build" -v "${v_dir}:/opt/v" \
```

Comment:
`build -> vbuild` should be an environment variable

Reply:
Implemented, vbuild is now passed via vtools via this PR: https://github.com/vectorizedio/vtools/pull/39